class/defaults.yml

```
@@ -90,7 +90,7 @@ parameters:
           hard:
             requests.cpu: 1000m
```

might make sense to bump CPU requests also then:

```suggestion
            requests.cpu: 4
```

---

makes sense, yes